app/models/sub_service_request.rb

```diff
@@ -24,12 +24,8 @@ class SubServiceRequest < ApplicationRecord
 
   audited
 
-  before_create :set_protocol_id
-  after_save :update_org_tree
-  after_save :update_past_status
-
   belongs_to :service_requester, class_name: "Identity", foreign_key: "service_requester_id"
-  belongs_to :owner, :class_name => 'Identity', :foreign_key => "owner_id"
+  belongs_to :owner, :class_name => 'Identity', :foreign_key => "owner_id", optional: true
```

Comment:
Not required right now, but required in Rails 6. Owner isn't assigned until later by an Admin